frontend/app/utils/accordion.component.js

```diff
@@ -30,14 +30,16 @@ export default class Accordion extends React.Component {
 
             return (
               <div key={index}>
-                <a
+                <div
```

Comment:
Might be better to use `<button className="unstyled">`. It's easier to get accessibility right by using `<button>` than by `<div role="button">`

See https://github.com/JustUtahCoders/utahexpungements.org/blob/92ec0e5ca1433231e0d46037175031192d4942eb/frontend/styleguide.js#L146

Reply:
Agreed! I’ll make the change later today.